src/prompts.ts

```diff
@@ -107,3 +95,192 @@ Make sure each affected file is summarized and it's part of the returned JSON.
     schema,
   })) as PullRequestSummary;
 }
+
+export type AIComment = {
+  file: string;
+  start_line: number;
+  end_line: number;
+  highlighted_code: string;
+  header: string;
+  content: string;
+  label: string;
+};
```

Comment:
Consider using zod schema for `AIComment` type definition instead of a plain type. This would allow runtime validation of the comment structure and provide consistent validation with the schema defined below:
```typescript
export const AICommentSchema = z.object({
  file: z.string(),
  start_line: z.number(),
  end_line: z.number(),
  highlighted_code: z.string(),
  header: z.string(),
  content: z.string(),
  label: z.string()
});

export type AIComment = z.infer<typeof AICommentSchema>;
```